reference/markup/checkbox-inline-no-js.html

```diff
@@ -9,6 +9,7 @@
 		name="checkboxes-new-inline--1"
 		checked="checked"
 		value=""
+		tabindex="0"
```

Comment:
It is if someone set's visibility to "hidden". It also isn't _hurting_ anything, so even if it isn't "needed" it's better safe than sorry. If the implementor knows enough to make the call to not put it in their code, that's fine. I guess it's like buckling your seat-belt. 99% of the time, it's a waste of time...